### PR TITLE
fix(oauth2): creating oauth2 via django management command

### PR DESCRIPTION
**Pull request by burnedikt**
this replaces the old logic where we injected python code into the django admin shell. Instead, now we're uploading a tested and safe admin command which will sooner or later be inclued with the backend image per default so that the fix becomes obsolete in this repository

### CODE REVIEW

**Comment by burnedikt** [comment]
fixes #35 due to more forgiving nature of the management command

**Comment by burnedikt** [comment]
decided to not merge this and stick with the old way of creating oauth2 clients until the backend supports a dedicated command at which point we can revisit this.

File: tasks/install_oauth_client_helper.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+- name: Copy OAuth2 Client Registration Script to server
```
Comment (LukasReindlGit) [inline tasks/install_oauth_client_helper.yml]: file raus

File: files/register_oauth_client.py [hunk 2]
```diff
@@ -0,0 +1,106 @@
+import json
```
Comment (LukasReindlGit) [inline files/register_oauth_client.py]: file can be removed

**Pull request closed without merging**